### PR TITLE
studynotekrwiki group permission changes per req

### PR DESCRIPTION
**Pull request by Reception123**
https://phabricator.miraheze.org/T290

### CODE REVIEW

**Comment by Reception123** [comment]
@Southparkfan @revi @JohnFLewis

File: LocalSettings.php [hunk 1]
```diff
@@ -2024,7 +2031,11 @@
 			'supervisor',
 			'wikifounder',
 		),
+		'+studynotekrwiki' => array(
+			'voter',
```
Comment (JohnFLewis) [inline LocalSettings.php]: Is user right right not exist.

**Comment by Reception123** [comment]
What do you mean? (The user right is created in $WgGroupPermissions)

**Comment by JohnFLewis** [comment]
I can't see it. I see a voter user *group* but I don't see a user *right*. See examples for the user protection level, the user group has the right 'user' which is used to define who has access.

**Comment by Reception123** [comment]
Add 'voter'

**Pull request merged**